File: CarRentalSystem/CarRentalSystem.Entities/Concrete/Car.cs
using CarRentalSystem.Core.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace CarRentalSystem.Entities.Concrete
{
    public class Car : IEntity
    {
        public int Id { get; set; }
        public int BrandId { get; set; }
        public int ColorId { get; set; }
        public string Name { get; set; }
        public int ModelYear { get; set; }
        public int DailyPrice { get; set; }
        public string Description { get; set; }
        public virtual Brand Brand { get; set; }
        public virtual Color Color { get; set; }
    }
}

File: CarRentalWebSystem/src/app/models/Car.ts
import { Brand } from './Brand';
import { CarImages } from './CarImages';
import { Color } from './Color';

export class Car {
  id: number;
  name: string;
  modelYear: number;
  dailyPrice: string;
  description: string;
  carImages: CarImages[];
  brand: Brand;
  color: Color;
}
